common/src/models/error.rs

```diff
@@ -30,114 +31,101 @@ impl Display for ApiError {
-            ),
-            ErrorKind::ProjectCorrupted => (
-                StatusCode::BAD_REQUEST,
-                "Tried to get project into a ready state for deletion but failed. Please reach out to Shuttle support for help."
-            ),
-            ErrorKind::ProjectHasResources(resources) => {
-                let resources = resources.join(", ");
-                return Self {
-                    message: format!("Could not automatically delete the following resources: {}. Please reach out to Shuttle support for help.", resources),
-                    status_code: StatusCode::INTERNAL_SERVER_ERROR.as_u16(),
-                }
-            }
-            ErrorKind::InvalidProjectName(err) => {
-                return Self {
-                    message: err.to_string(),
-                    status_code: StatusCode::BAD_REQUEST.as_u16(),
-                }
-            }
-            ErrorKind::InvalidOperation => (StatusCode::BAD_REQUEST, "The requested operation is invalid"),
-            ErrorKind::ProjectAlreadyExists => (StatusCode::BAD_REQUEST, "A project with the same name already exists"),
-            ErrorKind::OwnProjectAlreadyExists(message) => {
-                return Self {
-                    message,
-                    status_code: StatusCode::BAD_REQUEST.as_u16(),
-                }
-            }
-            ErrorKind::InvalidCustomDomain => (StatusCode::BAD_REQUEST, "Invalid custom domain"),
-            ErrorKind::CustomDomainNotFound => (StatusCode::NOT_FOUND, "Custom domain not found"),
-            ErrorKind::CustomDomainAlreadyExists => (StatusCode::BAD_REQUEST, "Custom domain already in use"),
-            ErrorKind::Unauthorized => (StatusCode::UNAUTHORIZED, "Unauthorized"),
-            ErrorKind::Forbidden => (StatusCode::FORBIDDEN, "Forbidden"),
-            ErrorKind::NotReady => (StatusCode::INTERNAL_SERVER_ERROR, "Service not ready"),
-            ErrorKind::DeleteProjectFailed => (StatusCode::INTERNAL_SERVER_ERROR, "Deleting project failed"),
-            ErrorKind::CapacityLimit => (StatusCode::SERVICE_UNAVAILABLE, "Our server is at capacity and cannot serve your request at this time. Please try again in a few minutes."),
+        let status = match kind {
+            ErrorKind::Internal => StatusCode::INTERNAL_SERVER_ERROR,
+            ErrorKind::KeyMissing => StatusCode::UNAUTHORIZED,
+            ErrorKind::ServiceUnavailable => StatusCode::SERVICE_UNAVAILABLE,
+            ErrorKind::KeyMalformed => StatusCode::BAD_REQUEST,
+            ErrorKind::BadHost => StatusCode::BAD_REQUEST,
+            ErrorKind::UserNotFound => StatusCode::NOT_FOUND,
+            ErrorKind::UserAlreadyExists => StatusCode::BAD_REQUEST,
+            ErrorKind::ProjectNotFound(_) => StatusCode::NOT_FOUND,
+            ErrorKind::ProjectNotReady => StatusCode::SERVICE_UNAVAILABLE,
+            // "not ready" is matched against in cargo-shuttle for giving further instructions on project deletion
```

Comment:
Could you move this comment up to the ProjectNotReady error message?

Reply:
Sorry, by error message I meant the corresponding `#[error()]` annotation in the enum definition